### PR TITLE
Add Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by ntzwilly**
## Description: 

- Create a class Person with the following:
    - Instance vars: @id, @name, and @age.
    - Constructor with name, age, and parent_permission as parameters. name and parent_permission are optional and have 
    - default values of "Unknown" and true.
    - Getters for @id, @name, and @age.
    - Setters for @name and @age.
    - Private method of_age? that returns true if @age is greater or equal to 18 and false otherwise.
    - Public method can_use_services? that returns true if person is of age or if they have permission from parents.
- Create class Student with the following:
     - Inherits from Person.
     - Constructor extends the parent's constructor by adding @classroom and a parameter for it.
     - Method play_hooky that returns "¯\(ツ)/¯".
- Create a class Teacher with the following:
     - Inherits from Person.
     - Constructor extends the parent's constructor by adding @specialization and a parameter for it.
     - Override can_use_services? so it always returns true.
     - Each class should be saved in a separate file.

### CODE REVIEW

**Comment by ArthurGC** [comment]
# INVALID SUBMISSION 🔴 

Hi @ntzwilly 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a descriptive README.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+source 'https://rubygems.org'
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+# gem "rails"
```
Comment (arslanbisharat) [inline Gemfile]: - [x] Please remove commented code.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: Gemfile (hunk 1 above)
Comment (ntzwilly) [reply Gemfile]: Thank you, I removed the commented code

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**